homeassistant/components/openweathermap/const.py

```diff
@@ -55,6 +55,7 @@
 ATTR_API_FORECAST_PRECIPITATION_PROBABILITY = "precipitation_probability"
 ATTR_API_FORECAST_PRESSURE = "pressure"
 ATTR_API_FORECAST_TEMP = "temperature"
+ATTR_API_FORECAST_HUMIDITY = "humidity"
```

Comment:
Please remove the unrelated changes to openweathermap. Only touch one integration per PR, unless doing some major refactoring of the project.

Reply:
removed, will redo in a new PR